tests/inverted-index-tests.js

```diff
@@ -176,6 +201,16 @@ describe('InvertedIndex.searchIndex', () => {
 });
 
 describe('inverted index api', () => {
+  describe('/api/search without index created', (done) => {
+    it('Should return an error message when trying to search without creating an index first', (done) => {
+      const request2 = supertest(app);
+      request2
+        .post('/api/search')
+        .send(['this is the third world secod of all'])
```

Comment:
Thank you. I think it doesn't matter. the search terms served it's purpose and it's passing